website/data/docs_detailed_categories.yml

```diff
@@ -65,6 +65,7 @@
   docs:
     - docs/agent/index.html
     - docs/agent/autoauth/index.html
+    - docs/agent/caching/index.html
```

Comment:
Will there be a sidebar link too?

Reply:
Do you know how to add it to the sidebar?

Reply:
https://github.com/hashicorp/vault/tree/master/website#updating-navigation